docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,57 @@
+
+### How are Instruments Created?
+
+LORIS instruments can be implemented in one of 2 formats: **LINST** or **PHP**.
+
+**LINST** is a LORIS in-house form format which is generated and edited by the LORIS Instrument Builder module. The LINST format allows for the creation of basic forms but can be complemented by a *score* file to allow for automated scoring and a *rules* file to accommodate for automatically checked rules.
+
+**PHP** instruments are forms coded manually by a developer in the PHP language. These instruments can also have rules and scoring defined.
+
+Choosing the format in which and instrument is coded is necessary before beginning the process. PHP instruments can easily be converted to LINST but the opposite is not true. Once an instrument is implemented in LINST format, transforming it to PHP is not trivial. Here are a few bullet points that can help you make your decision.
+
+- ***LINST***
+ - *Pros:*
+     - Easy to implement (LORIS Instrument Builder)
+     - Ideal for small instruments
+     - Less prone to bugs in the long term 
+     - Can be combined with rules and scoring 
+ - *Cons:*
+     - Rules and scoring logic can be limited in complex cases
+     - Rules and scoring might still require a developer
+     - Does not offer flexibility (default LORIS form elements are used)
+- ***PHP***
+ - *Pros:*
+     - Very flexible as form elements can be overwritten, implemented or customized
+     - allows for more complex validation and scoring
+     - Can easily be converted back to LINST
+ - *Cons:*
+     - Requires a developer
+     - Significant maintenance required over time to adapt to new LORIS releases
+     - less stable as some elements might change get deprecated over time
+
+### How is Instrument Data Stored?
+
+The data entered by users in an instrument form can be stored in 2 different formats: **SQL table** or **JSON string**. 
+
+JSON data storage makes instrument installation easier by removing the need for table creation in the database and reducing maintenance efforts in the long term. However, SQL storage allows for data amendments to be done directly through SQL commands without the need to parse the JSON data using a tool.
+
+The JSON data string is stored in the `Data` column of the `flag` table. If the SQL mode is enabled, a new table must be created and data is stored into the dedicated instrument table.
+
+>_**Note:** The support for JSON data storage is a new implementation that is meant to deprecate the use of SQL tables in the long term but is currently offered as an alternative only._
```

Comment:
```suggestion
>_**Note:** The support for JSON data storage is a new implementation that is meant to deprecate the use of SQL tables in the long term but is currently offered as an alternative only._
```
so... is it not fully supported?  can we just say it's in Beta testing? 
Are there any modules it doesn't automatically work with like Data Dictionary / DQT?

Reply:
as far as I know all modules support it right now. I would like to stay clear from offering as beta because projects will avoid using it if its labeled as such. It has been in beta since 23 and I think at this stage its actually an alternate

Reply:
Ok makes sense.